docs/index.md

```diff
@@ -1,18 +1,29 @@
----
-# generated by https://github.com/hashicorp/terraform-plugin-docs
```

Comment:
so this one won't be generated anymore ?

Reply:
Yes it'll come from the template file (maybe there's a way to embed the README in the template)